pyanaconda/modules/payloads/source/utils.py

```diff
@@ -29,6 +29,19 @@
 log = get_module_logger(__name__)
 
 
+def is_tar(url):
+    """Is the given URL a path to the tarball?
+
+    :param url: a string with URL
+    :return: True or False
+    """
+    if not url:
+        return False
+
+    tar_suffixes = (".tar", ".tbz", ".tgz", ".txz", ".tar.bz2", "tar.gz", "tar.xz")
```

Comment:
Dots are missing for the `tar.gz` and `tar.xz` suffixes? Otherwise http://my/tatar.gz would be considered a tarball.

Reply:
It's consistent with the previous code, though...

Reply:
Good catch! I have added a new commit with the fix and additional tests.